pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -483,17 +381,70 @@ func equalForOverlay(a1 []string, a2 []string) bool {
 	return true
 }
 
-func possibleIndicesForID(containerID string, mapping map[int]map[string]struct{}) []int {
-	var possibles []int
-	for index, ids := range mapping {
+func (se *StandardSecurityPolicyEnforcer) possibleIndicesForID(containerID string) []int {
+	var possibleIndices []int
+	for index, ids := range se.ContainerIndexToContainerIds {
 		for id := range ids {
 			if containerID == id {
-				possibles = append(possibles, index)
+				possibleIndices = append(possibleIndices, index)
 			}
 		}
 	}
+	return possibleIndices
+}
 
-	return possibles
+// EnforceMountPolicy for StandardSecurityPolicyEnforcer validates various
+// default mounts injected into container spec by GCS or containerD
+func (se *StandardSecurityPolicyEnforcer) EnforceMountPolicy(sandboxID, containerID string, spec *oci.Spec) (err error) {
+	defaultAllowedMounts, err := getDefaultMountConstraints(sandboxID, spec)
+	if err != nil {
+		return err
+	}
+	possibleIndices := se.possibleIndicesForID(containerID)
+	for _, specMnt := range spec.Mounts {
+		mountOk := false
+		// first check against default mounts
+		for _, mountConstraint := range defaultAllowedMounts {
+			if err = mountConstraint.validate(specMnt); err == nil {
+				mountOk = true
+				break
+			}
+		}
+
+		// no need to check further if the mount is among the default ones
+		if mountOk {
```

Comment:
Might be clearer to instead put a label on the outside loop, and do `continue <label>` instead of `break` above, then you can remove this statement.

Reply:
done